### ISSUE TITLE
runtime error with latest Aurelia skeleton (beta.1.2.0)

### ISSUE THREAD

**Issue by soarer25** [aurelia-ui-toolkits/aurelia-materialize-bridge#145]
Hi there,
I upgraded to the latest version of Aurelia and downloaded the skeleton app.
I added this package and I now get a 404 error on loading ...

Failed to load resource: the server responded with a status of 404 (Not Found)
Uncaught (in promise) Error: XHR error (404 Not Found) loading http://localhost:9000/jspm_packages/npm/aurelia-materialize-bridge@0.2.1/colors/md-colors.html.js(…)

Any ideas where the problem lies ? - the package was working perfectly for me yesterday and this only occurred after the upgrade to the latest Aurelia version

Cheers
    Garrett

**Comment by Thanood** [comment]
Thanks for mentioning, @soarer25.
Something has changed in the way the latest Aurelia loads resources. We need to investigate further. Someone on the KendoUI bridge brought up a (maybe related) [issue on that topic](https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/issues/430).

The only thing I can say for sure now is that ```FrameworkConfig``` and ```globalResources``` has changed, which we use internally. I think that this behaviour is not intended but I'd like to make really sure it isn't before I do something - it would f.i. be easy to create a component with both html and js parts out of md-colors.

**Comment by Thanood** [comment]
I think we've figured it out. I will try and deliver a patch release today.

As a workaround, you could edit ```config-builder.js``` of the bridge to use the text plugin with md-colors and md-well:

```javascript
  useColors() : ConfigBuilder {
    this.globalResources.push('./colors/md-colors.html!text');
    return this;
  }
```

and

```javascript
  useWell(): ConfigBuilder {
    this.globalResources.push('./well/md-well.html!text');
    return this;
  }
```

**Comment by soarer25** [comment]
​Great stuff - thanks a million for that!​

**Comment by Thanood** [comment]
0.2.2 just released, should fix this issue

**Comment by Thanood** [comment]
Reopening since I'm still not sure if this is a bug or a feature (strict SystemJS would be the feature).
https://github.com/aurelia/loader-default/issues/33

**Comment by Thanood** [comment]
Waiting for this to be merged: https://github.com/aurelia/framework/pull/383

**Comment by Thanood** [comment]
The PR mentioned above is released in aurelia-framework beta-1.2.1.
I will have to revert the workaround before it's usable.
